src/components/Legend.js

```diff
@@ -116,16 +126,18 @@ class Legend extends Component {
       uuid,
       iframeModeActive
     } = this.props;
-    const isMobile = !iframeModeActive && width < MOBILE_BREAKPOINT ? true : false;
+    const isMobile =
+      !iframeModeActive && width < MOBILE_BREAKPOINT ? true : false;
```

Comment:
Zelfde, `isMobile = !iframeModeActive && width < MOBILE_BREAKPOINT;` komt op hetzelfde neer.

Reply:
Deze is inderdaad identiek. Ik ga hem aanpassen.